Gemfile.lock

```diff
@@ -109,7 +109,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
-    rack (2.1.4)
+    rack (1.6.13)
```

Comment:
Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368)